src/Core/Extensible.php

```diff
@@ -18,7 +17,18 @@
  */
 trait Extensible
 {
-    use CustomMethods;
+    use CustomMethods {
```

Comment:
Unfortunately getExtensionInstances isn't a method on CustomMethods. It'll need refactoring.